docs/pre.sh

```diff
@@ -1,41 +1,42 @@
 #!/usr/bin/env bash
 
 ## Create modules pages
-for D in ../x/*; do
+for D in x/*; do
   if [ -d "${D}" ]; then
-    MODDOC=build/modules/$(echo $D | awk -F/ '{print $NF}')
+    MODDOC=docs/docs/build/modules/$(echo $D | awk -F/ '{print $NF}')
```

Comment:
why is this an issue? docs/docs doesnt exist anymore

Reply:
Apologies, this should just be for 0.50 and 0.47